packages/legend-query/src/components/QueryBuilderExplorerPanel.tsx

```diff
@@ -833,11 +833,13 @@ export const QueryBuilderExplorerPanel = observer(
     };
 
     useEffect(() => {
+      explorerState.initState.setMessage('Fetching mapped properties');
```

Comment:
could we move this inside the method where we call `modelCoverage`. 
adding in this useEffect adds an unnecessary watch on initState here.